auslib/test/admin/views/test_dockerflow.py

```diff
@@ -31,7 +31,8 @@ def testHeartbeatWithException(self):
             cr.side_effect = Exception("kabom!")
             # Because there's no web server between us and the endpoint, we receive
             # the Exception directly instead of a 500 error
-            self.assertRaises(Exception, self.client.get, "/__heartbeat__")
+            ret = self.client.get("/__heartbeat__")
+            self.assertEqual(ret.status_code, 502)
             self.assertEqual(cr.call_count, 1)
```

Comment:
It's also a good idea to check "ret.data" to make sure it contains the expected information in the body. You can see an example of this in https://github.com/mozilla/balrog/blob/8f8057168b0ab13177468cac93b85af4cfcb2472/auslib/test/admin/views/test_permissions.py#L255.